### ISSUE TITLE
Portainer endpoint is not reachable

### ISSUE THREAD

**Issue by apigban** [apigban/enthalpy#15]
Error 502:
```
{"log":"{\"level\":\"error\",\"ts\":1663702008.7740316,\"logger\":\"http.log.access\",\"msg\":\"handled request\",\"request\":{\"remote_ip\":\"XXXXXXXX\",\"remote_port\":\"5719\",\"proto\":\"HTTP/2.0\",\"method\":\"GET\",\"host\":\"portainer.avogadro.apigban.com\",\"uri\":\"/\",\"headers\":{\"Accept\":[\"text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9\"],\"Sec-Fetch-Mode\":[\"navigate\"],\"Sec-Fetch-Dest\":[\"document\"],\"Accept-Encoding\":[\"gzip, deflate, br\"],\"Sec-Ch-Ua\":[\"\\\"Google Chrome\\\";v=\\\"105\\\", \\\"Not)A;Brand\\\";v=\\\"8\\\", \\\"Chromium\\\";v=\\\"105\\\"\"],\"Sec-Fetch-Site\":[\"none\"],\"Cookie\":[],\"Sec-Ch-Ua-Mobile\":[\"?0\"],\"Upgrade-Insecure-Requests\":[\"1\"],\"User-Agent\":[\"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36\"],\"Cache-Control\":[\"max-age=0\"],\"Sec-Ch-Ua-Platform\":[\"\\\"Windows\\\"\"],\"Sec-Fetch-User\":[\"?1\"],\"Accept-Language\":[\"en-US,en;q=0.9\"]},\"tls\":{\"resumed\":false,\"version\":772,\"cipher_suite\":4865,\"proto\":\"h2\",\"server_name\":\"portainer.avogadro.apigban.com\"}},\"user_id\":\"\",\"duration\":0.096273154,\"size\":0,\**"status\":502,\**"resp_headers\":{\"Server\":[\"Caddy\"]}}\n","stream":"stderr","time":"2022-09-20T19:26:48.774134271Z"}
```

**Comment by apigban** [comment]
e932a5488880832f2284c4e91d994f0a66014ccf solves portainer endpoint access

![image](https://user-images.githubusercontent.com/2720851/191349846-60833c7b-f91e-4023-9d94-7e0d34ddaffd.png)

**Comment by apigban** [comment]
Merged: 47957b9998a31efc2f601e25d8063c04899f3744
